src/dynamixel/servos/servo.hpp

```diff
@@ -172,6 +172,20 @@ namespace dynamixel {
                 return reg_goal_position(id, pos);
             }
 
+            static inline InstructionPacket<protocol_t> reg_multi_turn_goal_position_angle(typename Servo<Model>::protocol_t::id_t id, double rad)
+            {
+                double deg = rad * 57.2958;
+                if (!(deg >= -92160 && deg <= 92160))  // 256 * 360
+                    throw errors::ServoLimitError(
+                        id,
+                        -92160.0 * 0.01745,
+                        92160.0 * 0.01745,
+                        rad
+                        );
+                typename ct_t::goal_position_t pos = deg * 11.37778; // * 360 / 4096
```

Comment:
```suggestion
                typename ct_t::goal_position_t pos = deg * 11.37778; // * 4096/360
```

resolution [pulse/rev] / 360 deg にすると見やすい

Reply:
対応しました